jipdate.py

```diff
@@ -191,7 +196,7 @@ def get_jira_issues(jira, exclude_stories, epics_only, all_status, filename,
         f.write("# Header: %s\n" % issue.fields.summary)
         f.write("# Type: %s\n" % issue.fields.issuetype)
         f.write("# Status: %s\n" % issue.fields.status)
-        f.write("No updates since last week.\n\n")
+        f.write("# No updates since last week.\n\n")
```

Comment:
Yes, like Christoffer says, an update with "No updates" is still a message to the members (basically it tells them that we haven't forgot about the issue, but haven't had time to look into it).